composer.json

```
@@ -12,7 +12,7 @@
         "beberlei/doctrineextensions": "^1.0",
         "caseyamcl/guzzle_retry_middleware": "^2.7.1",
         "doctrine/doctrine-orm-module": "^5",
-        "doctrine/orm": "~2.11.0",
+        "doctrine/orm": "2.19.5",
```

Any issue with expanding this constraint?

```suggestion
        "doctrine/orm": "^2.19.5",
```

---

Oops, hadn't meant to remove that. Is `^` correct though...that could eventually give us 2.20, 2.21 etc, while `~` will keep us on 2.19.

---

Doctrine follows semantic versioning so in "theory" we'd be okay on any 2.x version.